superdesk/utils.py

```diff
@@ -35,6 +35,32 @@ class SortOrder(Enum):
     desc = 2
 
 
+@unique
```

Comment:
@petrjasek it might be limiting. 

I will remove `@unique` decorator as it is trivial to remove.